valle/bin/trainer.py

```diff
@@ -1002,7 +1017,7 @@ def run(rank, world_size, args):
     )
     valid_dl = dataset.valid_dataloaders(valid_cuts)
 
-    if True:
+    if params.oom_check and params.start_epoch == 1:
```

Comment:
`if params.oom_check` let's check oom when training retart

Reply:
+1

Reply:
Updated the PR accoringly.